nemo_aligner/package_info.py

```diff
@@ -23,7 +23,13 @@
 VERSION = (MAJOR, MINOR, PATCH, PRE_RELEASE, DEV)
 
 __shortversion__ = ".".join(map(str, VERSION[:3]))
-__version__ = __shortversion__ + VERSION[3] + "." + ".".join(VERSION[4:])
+__version__ = __shortversion__
```

Comment:
WDYT about?
```suggestion
__version__ = ".".join(filter(None, map(str, VERSION)))
```
it's a little more succinct functionally

Reply:
Hm, the issue is that pre-release is joined w/o a dit, but the rest is. That makes it so clunky

Reply:
i missed that. good point